spec.html

```diff
@@ -23338,6 +23337,12 @@ <h1>The Global Object</h1>
   <emu-clause id="sec-value-properties-of-the-global-object">
     <h1>Value Properties of the Global Object</h1>
 
+    <emu-clause id="sec-global-this">
+      <h1>globalThis</h1>
+      <p>The initial value of `globalThis` is the value of the field of (the value of the field of the Realm Record named [[GlobalEnv]]) named [[GlobalThisValue]].
```

Comment:
this phrasing is *super awkward*. Any recommendations of how to do this cleaner without algorithm steps?

Reply:
```suggestion
      <p>The initial value of the `globalThis` property of the global object of a realm _realm_ is the value of `_realm_.[[GlobalEnv]].[[GlobalThisValue]]`.</p> 
```
That fudges the distinction between a realm and a Realm Record, so you could say ```... of a realm represented by the Realm Record _realm_ ...```, but I'm doubtful that'd be better.